src/pages/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
+const Statistics = () => {
+  const [contributorDetails, setContributorsDetails] = useState(null);
+
+  useEffect(() => {
+    axios
+      .get('https://api.github.com/repos/ijemmao/igbo_api/contributors')
+      .then((response) => {
+        setContributorsDetails(response.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
```

Comment:
Since we don't want to log errors into the console within a production environment, let's just pass in a noop like:

```js
.catch(() => {})
```

Reply:
I agree with you...it has been fixed